pkg/datapath/iptables/iptables.go

```diff
@@ -1199,7 +1250,7 @@ func (m *IptablesManager) InstallRules(ifName string) error {
 	localDeliveryInterface := getDeliveryInterface(ifName)
 
 	if err := m.installForwardChainRules(ifName, localDeliveryInterface, ciliumForwardChain); err != nil {
-		return fmt.Errorf("cannot install forward chain rules to %s: %s", transientChain.name, err)
+		return fmt.Errorf("cannot install forward chain rules to %s: %s", ciliumForwardChain, err)
```

Comment:
nit: We should really wrap the error here rather than formatting, `%w` at the end of the string.